src/main/java/Event.java

```diff
@@ -0,0 +1,21 @@
+public class Event extends Task {
+    
+    // Attributes
+    public String startTime;
+    public String endTime;
```

Comment:
I think you can use private/protected attributes instead of public for data encapsulation